### PR TITLE
Feature/stop depends on services

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description

Add the ability to instruct a service manager to stop other services before stopping it's own one
# Impacts

Services manager bro

### CODE REVIEW

**Comment by dwlfrth** [comment]
Review me

**Comment by dwlfrth** [comment]
Please review as soon as possible since it is required by something else :dancer:

File: bin/pfcmd-old.pl [hunk 1]
```diff
@@ -75,7 +75,8 @@ =head1 SYNOPSIS
 use constant {
     INSTALL_DIR        => '/usr/local/pf',
     JUST_MANAGED       => 1,
-    INCLUDE_DEPENDS_ON => 2,
+    INCLUDE_START_DEPENDS_ON => 2,
+    INCLUDE_STOP_DEPENDS_ON => 3,
```
Comment (jrouzierinverse) [inline bin/pfcmd-old.pl]: This should be 4 not 3.

Comment (jrouzierinverse) [inline bin/pfcmd-old.pl]: These are flags not enumerations.
It should be changes to use bit notation.

0b0000001
0b0000010
0b0000100

To make it more clear

File: lib/pf/cmd/pf/service.pm [hunk 2]
```diff
@@ -74,7 +74,8 @@ use pf::services;
 use List::MoreUtils qw(part any true all);
 use constant {
     JUST_MANAGED       => 1,
-    INCLUDE_DEPENDS_ON => 2,
+    INCLUDE_START_DEPENDS_ON => 2,
+    INCLUDE_STOP_DEPENDS_ON => 3,
```
Comment (jrouzierinverse) [inline lib/pf/cmd/pf/service.pm]: Same as above

**Comment by dwlfrth** [comment]
Re-review mew

**Pull request merged**